### DISCUSSION TITLE
Individual coloring of lines (background color) inside a code block

### DISCUSSION THREAD

**Discussion by GitHub2022** [asciidocfx/AsciidocFX#644] [General]
Hi, for teaching books it might be necessary to set individual lines in code block to bold or change the background color to yellow for example to show what changed between almost the same code instances. Is there any way? 

I tried already setting these things in the header: [source,json,subs="verbatim,quotes"] but still the preview doesnt show any *bold* text nor can i set the background color for specific words or lines.

**Comment by rahmanusta** [comment]
This is a asciidoctor related question. You may look at asciidoctor documentation and ask in their chat https://asciidoctor.zulipchat.com/
